invenio_mail/ext.py

```diff
@@ -49,23 +49,33 @@ def print_email(message, app):
 
 
 class InvenioMail(object):
-    """Invenio-Mail extension.
+    """Invenio-Mail extension."""
 
-    Mails are only printed to the stream if ``MAIL_SUPPRESS_SEND`` is ``True``.
+    def __init__(self, app=None, stream=None):
+        """Extension initialization.
 
-    :param app: Flask application object.
-    :param stream: Stream to print emails to. Defaults to ``sys.stdout``.
-    """
+        Mails are only printed to the stream if ``MAIL_SUPPRESS_SEND`` is
+        ``True``.
 
-    def __init__(self, app=None, stream=None):
-        """Extension initialization."""
+        :param app: Flask application object.
```

Comment:
What do you think about https://github.com/inveniosoftware/invenio-mail/pull/18/files#diff-85987f48f1258d9ee486e3191495582dR328?